subsys/bluetooth/common/Kconfig

```diff
@@ -114,7 +114,7 @@ config BT_BUF_EVT_RX_SIZE
 	default 255 if (BT_EXT_ADV && !(BT_BUF_EVT_DISCARDABLE_COUNT > 0))
 	# LE Read Supported Commands command complete event.
 	default 68
-	range 68 255
+	range 68 264
```

Comment:
What event can be larger than 255? The length field is 1 byte. Should this perhaps be ACL_RX_SIZE?

Reply:
The requesting name fails after calling bt_br_discovery_start, I compared with old codes, and changing this field to 264 can fix it.

Reply:
The Host shall be able to accept HCI Event
packets with up to 255 octets of data excluding the HCI Event packet header.

Can you please figure out why 264 fixes it when no event should be larger than 255 bytes?
Is a packet going to the wrong buffer pool perhaps?

Reply:
Thanks. I cannot re-produce it now, I have removed the code changes. If I re-produce it again, I will create another PR.